examples/Database source examples/aws_dynamo_db_authentication/connector.py

```diff
@@ -0,0 +1,97 @@
+# Refer boto3 docs (https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/dynamodb.html)
```

Comment:
Please add Java comments similar to other examples. Also, add an example of what the source and destination look like. Please refer to other examples

Reply:
Done.